docs/source/algorithms/MaskDetectors-v1.rst

```diff
@@ -309,7 +367,88 @@ Output
   Detector 102 is masked: False
   Detector 103 is masked: True
   Detector 104 is masked: False
+  
+Example 5: Specifying a masking range
+#####################################
+
+.. testcode:: ExMaskInRange
+
+  # Create a data workspace
+  ws = CreateSampleWorkspace()
+  # Mask 3 detectors using the masking range
+  MaskDetectors(ws, StartWorkspaceIndex=2, EndWorkspaceIndex=4)  
 
+  # Check masking of first 6 detectors
+  for ind in xrange(0,6):
+    det = ws.getDetector(ind)
+    print 'Detector',det.getID(),'is masked:',det.isMasked()
```

Comment:
print() under python 2.7 works but prints brackets too. 
Lethal for doctests which run under 2.7